### PR TITLE
Neti karam dosa

### PR DESCRIPTION
**Pull request by vinaygrp1**

### CODE REVIEW

**Comment by vinaydms** [comment]
> _ No description provided. _

avasrama extra karam

**Comment by vinaygrp1** [comment]
> > _ No description provided. _
> 
> avasrama extra karam

it's necessary .

File: menu.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 dosa
-karam
-oil
\ No newline at end of file
+extra karam
```
Comment (vinaydms) [inline menu.txt:2]: do we really require extra karam ?

Comment (vinaygrp1) [reply menu.txt:2]: yes if extra karam extra tasty

Comment (vinaydms) [reply menu.txt:2]: yaa, that's nice

**Review by vinaydms — APPROVED** [review]
yes Neti karam is good

**Pull request merged**